addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -256,6 +256,72 @@ QUnit.test('chat window new message: fold', async function (assert) {
     );
 });
 
+QUnit.test('state should not be changed as "clicked" when author name is clicked', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
+        state: 'open',
+    });
+    this.data['mail.message'].records.push({
+        body: "not empty",
+        channel_ids: [20],
+    });
+    await this.start({
+        env: {
+            browser: {
+                innerWidth: 900,
+            },
+        },
+    });
+    assert.containsOnce(
+        document.body,
+        '.o_ChatWindow',
+        "should have opened one chat window initially"
+    );
+    document.querySelector(`.o_Message_authorName`).click();
+    await nextAnimationFrame();
+    assert.doesNotHaveClass(
+        document.querySelector(`.o_Message`),
+        'o-clicked',
+        "message should not contain 'o-clicked' class"
+    )
```

Comment:
Do not repeat what the assert does technically in the message, this doesn't help understand it.
A good single assert test would usually have something very close to the test title in the assert comment.